xc/device.ml

```diff
@@ -2227,7 +2256,9 @@ module Backend = struct
           Dm_Common.vnc_socket_path domid;
           (qmp_event_path domid);
           (qmp_libxl_path domid);
-        ] |> List.iter Socket.Unix.rm
+        ] |> List.iter Socket.Unix.rm;
```

Comment:
Is this safe or could throw an exception and prevent the cleanup below?

Reply:
These are simple functions that just return the path - I think it is safe